src/stores/data.js

```diff
@@ -76,7 +76,7 @@ function getRow(dataInfo, key) {
     symbol: dataInfo.symbol,
     owner: dataInfo.owner_addr,
     lastroundapy: dataInfo.lastRoundAPY,
-    // doubleapy: parseFloat(dataInfo.apy * 2).toFixed(3), // TODO: enable after discussion around it
+    doubleapy: parseFloat(dataInfo.apr * 2).toFixed(3),
```

Comment:
This calculation should be double the weekly yield and then compounded across 52 weeks to get the APY

Reply:
`convertWPRtoAPY(dataInfo.apr * 2)`

Reply:
changed 👍🏽 thanks